pifi/driver.py

```diff
@@ -0,0 +1,85 @@
+class APA102Driver():
```

Comment:
FYI changes I needed to get your patches to work: https://gist.github.com/dasl-/4bc1c5be6c715e8c8a1c191eb3fb477d

Reply:
Done